apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.html

```diff
@@ -0,0 +1,29 @@
+<pa-modal-dialog>
+  <pa-modal-title>
+    <h1>{{ 'account.delete_account_confirm_title' | translate }}</h1>
+  </pa-modal-title>
+  <pa-modal-content>
+    <div>
+      <label>{{ 'account.delete_account_confirm' | translate:{ account: (account | async)?.title } }}</label>
+      <pa-checkbox [(value)]="deleteAccount">{{ 'generic.yes' | translate }}</pa-checkbox>
```

Comment:
We agreed with Anna to replace the yes by just a checkbox with a label like "I want to delete the [name] account and all the data on it." (I see she didn't updated the figma file, but she seemed to like this idea in the comment)

Reply:
Fixed